# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/layout.js
# Commit: 3cfe5bd5d084
# Full Hash: 3cfe5bd5d084baf18f5d95b0e879f6fc179fac34
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-06 04:09:04
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 76a965804210 -r 3cfe5bd5d084 accessible/tests/mochitest/layout.js
--- a/accessible/tests/mochitest/layout.js	Tue Oct 05 20:24:03 2021 +0000
+++ b/accessible/tests/mochitest/layout.js	Tue Oct 05 20:44:46 2021 +0000
@@ -280,26 +280,26 @@
 
   info(`DPR is: ${aDPR}`);
   isWithin(
+    xInCSS.value,
     x.value / aDPR,
-    xInCSS.value,
     1,
     "X in CSS pixels is calculated correctly"
   );
   isWithin(
+    yInCSS.value,
     y.value / aDPR,
-    yInCSS.value,
     1,
     "Y in CSS pixels is calculated correctly"
   );
   isWithin(
+    widthInCSS.value,
     width.value / aDPR,
-    widthInCSS.value,
     1,
     "Width in CSS pixels is calculated correctly"
   );
   isWithin(
+    heightInCSS.value,
     height.value / aDPR,
-    heightInCSS.value,
     1,
     "Height in CSS pixels is calculated correctly"
   );